### ISSUE TITLE
add speed and direction as plot quantities in plot.vpts()

### ISSUE THREAD

**Issue by adokter** [adokter/bioRad#424]

**Comment by peterdesmet** [comment]
If we do, we should definitely use a different colour scale, so these plots cannot be confused with dens plots.

**Comment by baptischmi** [comment]
sound nice, but then the two quantities (speed and direction) are plotted in separated vpts-plots, right? How about plotting wind-profit instead?
One can think of adding the wind speed and direction as barbs to these bird speed vpts-plots?

**Comment by adokter** [comment]
Yes @baptischmi, it would be adding an option so you could plot speed instead of bird density as the profile color, so typically you would make two plots

 To really do plots with wind we need functionality and conventions for how to add and name wind data - some of that is already present in hidden form. Same with wind profit, you could start a separate issue for those things

**Comment by adokter** [comment]
@bmvandoren as original requestor, any thoughts on a default color scale for this? Probably one of the viridis color scales would work well.

**Comment by bmvandoren** [comment]
I was thinking a viridis scheme as well

**Comment by CeciliaNilsson709** [comment]
Good idea, and good to have different default color scales for all three (dens, speed, direction) then, just to avoid confusion. 

We could also do a north/south component plot combining direction and speed into one plot. That would then need a more divergent color scale, and could be a pretty cool way of looking at for instance how "strong" of a migratory movement there is at different altitudes. East/west movements would of course not show up very well though, so that might be an issue.

**Comment by baptischmi** [comment]
@CeciliaNilsson709: something like > https://images.app.goo.gl/qZKg1ZDAaVjNtL3X7

**Comment by CeciliaNilsson709** [comment]
Hmm, I was thinking more calculating speed-direction vectors (similar to U and V wind), and showing it in the vpts plot so you can for instance see if fast, directed migration occurs at high altitude and slower, less directed movements at low altitudes etc.
